xbmc/pvr/channels/PVRChannel.cpp

```diff
@@ -61,6 +61,7 @@ CPVRChannel::CPVRChannel(bool bRadio /* = false */)
   m_iClientId               = -1;
   m_iClientEncryptionSystem = -1;
   UpdateEncryptionName();
+  m_iOrder = 0;
```

Comment:
nitpick: maybe we can put the new member after the last existing member (and add a blank line to visually separate methods from members.)